components/Trial/Form.js

```diff
@@ -98,6 +98,7 @@ const Form = props => {
   const [tokenType, setTokenType] = useState('EMAIL_CODE')
   const [showErrors, setShowErrors] = useState(false)
   const [autoRequestAccess, setAutoRequestAccess] = useState(false)
+  const [payload] = useState(props.payload)
```

Comment:
This is actually not necessary and could lead to bugs.

E.g. when navigating from Episode 1 to Episode 2 the pay note could potentially not be remounted and the old payload would persist.

Directly using `props.payload` should be fine because by the time the variation changes to thank you the form is no longer visible and the `requestAccess` mutation has already run with the correct variation.

Reply:
Thank you, got it.